src/commands/build.ts

```diff
@@ -85,30 +84,35 @@ export class BuildCommand implements Command {
 
   async run(options: CommandOptions, config: ProjectConfig): Promise<any> {
     // Defer dependency loading until this specific command is run
-    let build = require('../build/build').build;
-    const hasCLIArgumentsPassed = this.args.some(arg => {
-      return typeof options[arg.name] !== 'undefined';
-    });
+    let {build} = require('../build/build');
```

Comment:
bah you're right, the fact that typescript transpiles import restructuring always trips me up